smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSource.java

```diff
@@ -59,7 +66,74 @@ public KafkaSource(Vertx vertx, KafkaConnectorIncomingConfiguration config) {
         kafkaConfiguration.remove("broadcast");
         kafkaConfiguration.remove("partitions");
```

Comment:
The new attribute need to be removed from the kafkaConfiguration map. It avoids a warning.

Reply:
done